server/index.js

```diff
@@ -1,3 +1,7 @@
+// install heroku global `npm install -g heroku`
+// login heroku `heroku login`
+//From the top level of repo use command `git subtree push --prefix server heroku-backend master`
```

Comment:
we shouldn't need this once we setup heroku CD. You would generally only need heroku CLI in cases where you want to look at logs or handle migration issues.